cli/src/test/java/com/devonfw/tools/ide/commandlet/EditionGetCommandletTest.java

```diff
@@ -72,10 +72,7 @@ public void testEditionGetCommandletToolNotInstalled() {
     editionGet.tool.setValueAsString("az", context);
     editionGet.run();
     // assert
-    assertThat(context).log().hasEntries(IdeLogEntry.ofWarning("No installation of tool az was found."),
-        IdeLogEntry.ofInfo("The configured edition for tool az is az"),
-        IdeLogEntry.ofInfo("To install that edition call the following command:"),
-        IdeLogEntry.ofInfo("ide install az"));
+    assertThat(context).logAtProcessable().hasMessage("az");
```

Comment:
here it seems vice versa. The output in the test was like this:
```
14:20:56.527 [main] - INFO  - c.d.tools.ide.log.IdeSubLoggerSlf4j - PROCESSABLE:No installation of tool az was found.
14:20:56.527 [main] - INFO  - c.d.tools.ide.log.IdeSubLoggerSlf4j - PROCESSABLE:The configured edition for tool az is az
14:20:56.527 [main] - INFO  - c.d.tools.ide.log.IdeSubLoggerSlf4j - PROCESSABLE:To install that edition call the following command:
14:20:56.527 [main] - INFO  - c.d.tools.ide.log.IdeSubLoggerSlf4j - PROCESSABLE:ide install az
```

So IMHO something is still broken with the implementation.